lib/logidze/model.rb

```diff
@@ -195,6 +195,12 @@ def association(name)
     end
     # rubocop: enable Metrics/MethodLength
 
+    def log_size
+      return 0 unless log_data
```

Comment:
We can use safe-navigation here:

```ruby
log_data&.size || 0
```

WDYT?

Reply:
Would this be good enough for this case? (`nil.to_i -> 0`)
```ruby
log_data&.size.to_i
```